src/reaction_network.jl

```diff
@@ -186,36 +145,74 @@ emptyrn = @reaction_network empty
 # an empty network with random generated name
 emptyrn = @reaction_network
 ```
+
+ReactionSystems generated through `@reaction_network` are complete.
 """
+# macro reaction_network(args...)
+#     return :(complete(@network_component $(args... )))
+# end
 macro reaction_network(name::Symbol, ex::Expr)
-    make_reaction_system(MacroTools.striplines(ex); name = :($(QuoteNode(name))))
+    :(complete($(make_reaction_system(MacroTools.striplines(ex); name = :($(QuoteNode(name)))))))
```

Comment:
Here I just reimplements the two macros identically, until we can figure out how to use one set of implementations.